docs/pre-release/reference/config.md

```diff
@@ -110,3 +111,22 @@ clusters:
         - 1.2.3.4/32
   name: example-cluster
 ```
+
+#### Manager
+
+The `manager` key contains configuration for finding the `traffic-manager` that telepresence will connect to. It supports one key, `namespace`, indicating the namespace where the traffic manager is to be found
+
+Here is an example kubeconfig that will instruct telepresence to connect to a manager in namespace `staging`:
+
+```
+apiVersion: v1
+clusters:
+- cluster:
+    server: https://127.0.0.1
+    extensions:
+    - name: telepresence.io
+      extension:
+        manager:
+          namespace: staging
+  name: example-cluster
+```
```

Comment:
in addition to this should we add a `--namespace` argument to `telepresence connect`? that way users can switch back and forth without having to come back to this file

Reply:
I think that would be a good idea but not necessary for v0... so all to say, I think you can leave it out of the docs for now and then we can add it if we decide to add that feature.  

In general, I think it'd be nice if _all_ the config values could be set with the command line, I think some folks have requested that for timeouts and such.  (not saying you need to take that on, but it is definitely something we have wanted to do :) )

Reply:
ok, i think it makes sense to keep it out for now then :) especially if we might later want to come up with a way to do it for all configs